apmproxy/receiver_test.go

```diff
@@ -68,7 +68,7 @@ func TestInfoProxy(t *testing.T) {
 	}()
 
 	hosts, _ := net.LookupHost("localhost")
-	url := "http://" + hosts[0] + ":1234"
+	url := "http://" + hosts[1] + ":1234"
```

Comment:
`hosts[0]` yields the IPV6 address. However, due to the fact that IPV6 is not supported on the `ubuntu-*` runners (See: https://github.com/actions/runner-images/issues/668#issuecomment-624080758) the tests are failing.

Using `hosts[1]` instead yields the IPV4 address and fixes the test.